extras/exporters/gltf-exporter.js

```diff
@@ -350,13 +391,25 @@ class GltfExporter extends CoreExporter {
                 }
 
                 if (blendType === BLEND_NORMAL) {
-                    material.alphaMode = "BLEND";
+                    material.alphaMode = 'BLEND';
+                } else if (blendType === BLEND_NONE) {
+                    if (alphaTest !== 0) {
+                        material.alphaMode = 'MASK';
+                        material.alphaCutoff = alphaTest;
+                    } else {
+                        // It is a default value, so we don't need to write it
+                        // material.alphaMode = 'OPAQUE';
+                    }
```

Comment:
Personally, I would just delete the else clause here.

Reply:
Done